### PR TITLE
831665: Appearance UG for SfAccordion.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Created AppearanceUg for MAUI SfAccordion.

### CODE REVIEW

File: MAUI/Accordion/appearance.md [hunk 1]
```diff
@@ -0,0 +1,496 @@
+---
+layout: post
+title: Appearance | Accordion for Maui | Syncfusion
+description: Learn about Appearance support in Syncfusion Maui Accordion (SfAccordion) control and more details.
```
Comment (M-A32) [inline MAUI/Accordion/appearance.md:4]: Description: Learn about Appearance support in Syncfusion Maui Accordion (SfAccordion) control and more details.

File: MAUI/Accordion/appearance.md [hunk 2]
```diff
@@ -0,0 +1,496 @@
+---
+layout: post
+title: Appearance | Accordion for Maui | Syncfusion
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/appearance.md:3]: Here, title should be in small.

File: MAUI/Accordion/appearance.md [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+    var item = new AccordionItem();
+    item.IconColor = Color.Accent;
+    return item;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Visual State Manager
+
+The appearance of the `SfAccordion` can be customized using the following two `VisualStates`:
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/appearance.md:102]: Addressed the concerns provided.

File: MAUI/Accordion/appearance.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+ 
+The `SfAccordion` allows you to customize the position of the header icon in each accordion item using the `HeaderIconPosition` property. By default, the header icon position is set to `End`.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfAccordion x:Name="accordion" 
+                        HeaderIconPosition="Start" />
+{% endhighlight %}
+{% highlight c# %}
+accordion.HeaderIconPosition = Syncfusion.Maui.Expander.IconPosition.Start;
```
Comment (PiruthivirajM) [inline MAUI/Accordion/appearance.md:24]: Enum name -> ExpanderIconPosition

**Review by PiruthivirajM — COMMENTED** [review]
address review comments

File: MAUI/Accordion/appearance.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/Accordion/appearance.md]: Modified

File: MAUI/Accordion/appearance.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+        </syncfusion:SfAccordion>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="112 113 115 116" %}
+public partial class MainPage : ContentPage
+{
+    SfAccordion accordion;
+	public MainPage()
+	{
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Accordion/appearance.md]: modified

File: MAUI/Accordion/appearance.md [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+
+## Icon color customization
+
+The `SfAccordion` allows you to customize the color of the expander icon by using the `HeaderIconColor` property. By default, the `HeaderIconColor` is `Black`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3" %}
+<syncfusion:SfAccordion x:Name="accordion">
+    <syncfusion:SfAccordion.Items>
+        <syncfusion:AccordionItem HeaderIconColor="Accent"/>
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Accordion/appearance.md]: modified

File: MAUI/Accordion/appearance.md [hunk 7]
```diff
@@ -0,0 +1,318 @@
...
+    var item = new AccordionItem();
+    item.HeaderBackground = Color.Pink;
+    return item;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Icon color customization
+
+The `SfAccordion` allows you to customize the color of the expander icon by using the `HeaderIconColor` property. By default, the `HeaderIconColor` is `Black`.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Accordion/appearance.md]: Modified

**Review by PiruthivirajM — APPROVED** [review]
approved

**Pull request merged**